### PR TITLE
Update jest to 25.1

### PR DESCRIPTION
**Pull request by thboop**
Audit tools step is failing, lets up date the packages to address this

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "eslint-plugin-github": "^2.0.0",
     "eslint-plugin-jest": "^22.5.1",
     "flow-bin": "^0.115.0",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
```
Comment (ericsciple) [inline package.json]: is this PR causing the \"audit tools\" step to fail?

Comment (ericsciple) [reply package.json]: or was it failing before?

Comment (ericsciple) [reply package.json]: looks like it's failing before. Make sure not adding new problems :)

Comment (thboop) [reply package.json]: It was failing before, the intention was to fix it but i missed a few packages, pushing up more updates :)

**Pull request merged**